cfme/physical/provider/lenovo.py

```diff
@@ -32,6 +32,15 @@ class LenovoProvider(PhysicalProvider):
     settings_key = 'ems_lenovo'
     log_name = 'lenovo'
 
+    @property
+    def mgmt(self):
```

Comment:
**REQUIRED**
Why did you reimplement https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/provider.py#L120? `LenovoProvider` should inherit `mgmt` property from `BaseProvider` class.

Reply:
because ``mgmt`` property from ``BaseProvider`` doesn't provide neither hostname, username nor password, and I was inspired by `` VmwareCloudProvider`` https://github.com/ManageIQ/integration_tests/blob/b5338f9686db6acf9e9a4ce8c48cd2fe875d3f85/cfme/cloud/provider/vcloud.py#L56-L65